src/coreComponents/mesh/simpleGeometricObjects/Box.cpp

```diff
@@ -65,6 +65,13 @@ void Box::postProcessInput()
   LvArray::tensorOps::add< 3 >( m_boxCenter, m_max );
   LvArray::tensorOps::scale< 3 >( m_boxCenter, 0.5 );
 
+  // reordering min and max to fit former interface but so that user can input any of the two diagonals
```

Comment:
// ... any of four diagonals?

Maybe add a GEOS message to tell the user about the swap.